apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-wp-rest-image-sideload-controller.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * WP_REST_Image_Sideload_Controller file.
+ *
+ * @package full-site-editing
+ */
+
+/**
+ * Class WP_REST_Image_Sideload_Controller.
+ */
+class WP_REST_Image_Sideload_Controller extends WP_REST_Attachments_Controller {
+
+	/**
+	 * WP_REST_Image_Upload_Controller constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wp/v2';
+		$this->rest_base = 'media/image';
```

Comment:
Will this need to be updated?

Reply:
I would prefer if it mentioned our plugin name/slug as this is definitely nothing we want to make available for other use or risk collisions over super-generic naming.